R/dp_tags.R

```diff
@@ -373,6 +373,87 @@ get_property_tags <- function(property, proj_years = NULL) {
         func = function() get_default_art_mortality_rates(proj_years)
       )
     ),
+    art_15plus_num_percent = list(
```

Comment:
this looks a lot like *data* rather than code - how would you feel about moving it into something like a yaml file within `inst/` then reading it on package load?